apps/data-service-catalog/app/catalogs/[catalogId]/data-services/new/page.tsx

```diff
@@ -0,0 +1,42 @@
+import { Breadcrumbs, BreadcrumbType, PageBanner } from '@catalog-frontend/ui';
+import { getOrganization } from '@catalog-frontend/data-access';
+
+import { Params } from 'next/dist/shared/lib/router/utils/route-matcher';
+import { getTranslateText, localization } from '@catalog-frontend/utils';
+import { Organization } from '@catalog-frontend/types';
+import DataServiceForm from '../../../../../components/data-service-form';
+import { dataServiceToBeCreatedTemplate } from '../../../../../components/data-service-form/utils/data-service-initial-values';
+
+export default async function NewDataServicePage({ params }: Params) {
```

Comment:
Sjekke om man har skrivetilgang?

Reply:
Tenker det beste er å få inn noe lignende `withProtectedPage`, som du har i begrepskatalogen. Og helst da både for new og edit, pluss sjekk av lesetilgang på søkesida og detaljsida. Men det krever litt refaktorering, tar det i egen pr, issue her: #926